components/core/src/clp/GlobalMetadataDBConfig.hpp

```diff
@@ -48,18 +75,23 @@ class GlobalMetadataDBConfig {
 
 private:
     // Variables
+    static constexpr std::string_view cDefaultMetadataDbTypeName{"sqlite"};
+    static constexpr MetadataDBType cDefaultMetadataDbType{MetadataDBType::SQLite};
+    static constexpr std::string_view cDefaultMetadataDbHost{"127.0.0.1"};
     static constexpr int cDefaultMetadataDbPort{3306};
+    static constexpr std::string_view cDefaultMetadataDbName{"clp-db"};
+    static constexpr std::string_view cDefaultMetadataTablePrefix{"clp_"};
 
-    MetadataDBType m_metadata_db_type{MetadataDBType::SQLite};
+    MetadataDBType m_metadata_db_type{cDefaultMetadataDbType};
 
-    std::string m_metadata_db_host{"localhost"};
+    std::string m_metadata_db_host{cDefaultMetadataDbHost};
     int m_metadata_db_port{cDefaultMetadataDbPort};
-    std::string m_metadata_db_name;
+    std::string m_metadata_db_name{cDefaultMetadataDbName};
 
     std::string m_metadata_db_username;
     std::string m_metadata_db_password;
 
-    std::string m_metadata_table_prefix;
+    std::string m_metadata_table_prefix{cDefaultMetadataTablePrefix};
```

Comment:
I would put this after line 89 so it aligns with the order we fetch and validate algorithm. I also don't see why previously we put the table_prefix after username and password.